uber/model_checks.py

```diff
@@ -134,9 +134,14 @@ def attendee_money(attendee):
 
 def attendee_badge_range(attendee):
     if AT_THE_CON:
```

Comment:
Previously if you tried to enter a badge number which isn't even an integer, you'd get a stacktrace instead of a helpful error message.